integration/setup/common.go

```diff
@@ -351,5 +370,36 @@ func startup() {
 		} else {
 			zap.S().Infof("Compat system: port %d.", p)
 		}
+
+		// Set open telemetry tracer if jaeger endpoint is provided.
+		if *jaegerEndpointF != "" {
+			startupTracer(tb)
+		}
+	})
+}
+
+// startupTracer initializes open telemetry tracer that could be used in tests.
+func startupTracer(tb testing.TB) {
+	exp, err := jaeger.New(jaeger.WithCollectorEndpoint(jaeger.WithEndpoint(*jaegerEndpointF)))
+	if err != nil {
+		tb.Errorf("failed to create jaeger exporter: %v", err)
+	}
+
+	tp := tracesdk.NewTracerProvider(
+		tracesdk.WithBatcher(exp),
+		tracesdk.WithSampler(tracesdk.AlwaysSample()),
+		tracesdk.WithResource(resource.NewSchemaless(
+			semconv.ServiceNameKey.String("FerretDB"),
+		)),
+	)
+
+	// Register TracerProvider globally to use it by default
+	otel.SetTracerProvider(tp)
+
+	tb.Cleanup(func() {
```

Comment:
That runs when the test that first called `startup` exits, not when all tests exit

Reply:
Hmmm... Could you recommend me what I can use instead of this? I was looking for something like `teardown` similar to `startup`, but I don't see it.